api/axis.py

```diff
@@ -197,29 +216,37 @@ def update_licensepools_for_identifiers(self, identifiers):
         # k books are the identifiers in `remainder`. These books have
         # been removed from the collection without us being notified.
         for removed_identifier in remainder:
-            pool = removed_identifier.licensed_through
-            if not pool:
-                self.log.warn(
-                    "Was about to reap %r but no local license pool.",
-                    removed_identifier
-                )
-                continue
-            if pool.licenses_owned == 0:
-                # Already reaped.
-                continue
-            self.log.info(
-                "Reaping %r", removed_identifier
-            )
+            self.reap_license_pool_for(removed_identifier)
+
+    def reap_license_pool_for(self, identifier):
+        """Reflect the fact that the LicensePool associated with an Identifier
+        is no longer in the collection.
+        """
+        if not identifier:
+            return
 
-            availability = CirculationData(
-                data_source=pool.data_source,
-                primary_identifier=removed_identifier,
-                licenses_owned=0,
-                licenses_available=0,
-                licenses_reserved=0,
-                patrons_in_hold_queue=0,
+        pool = identifier.licensed_through
+        if not pool:
+            self.log.warn(
+                "Was about to reap %r but no local license pool.",
+                identifier
             )
-            availability.apply(pool, False)
```

Comment:
This is the bug that would have broken the reaper before. The second argument to `apply` is supposed to be a policy object, not a boolean.